src/main/app/claims/ccjModelConverter.ts

```diff
@@ -34,9 +45,48 @@ function convertPayBySetDate (draftCcj: DraftCCJ): Moment {
     ? draftCcj.payBySetDate.date.toMoment() : undefined
 }
 
+export function retrieveDateOfBirthOfDefendant (claim: Claim): DateOfBirth {
```

Comment:
Can be moved to either utility class or inside `Claim` . Missing unit tests for this as well.